Makefile

```diff
@@ -699,7 +699,7 @@ generate-gitignore:
 .PHONY: generate-images
 generate-images:
 	npm install --no-save --no-package-lock fabric imagemin-zopfli
-	node build/generate-images.js
+	node build/generate-images.js $(TAGS)
```

Comment:
Please explain what this is doing in `docs/content/doc/developers/hacking-on-gitea.en-us.md`.

Reply:
BTW that hacking doc is probably not the best place to talk about image customization, we should maybe move the paragraph to a page about customization.

Reply:
Hacking doc would be for anyone looking to change the logo as a contribution.

For someone who wants to change their _own_ logo for their instance, the instructions would be in https://docs.gitea.io/en-us/customizing-gitea/

Reply:
Yes, the `make generate-images` is meant for end users to build their custom logo, we should move the docs.

Reply:
Should be all set. Could use some help with a Chinese translation.

Reply:
I think chinese docs are already vastly out of date so I wouldn't worry about them. But sure, if someone's willing.